src/ui/server/helpers/get-application-by-reference-number-variables/index.ts

```diff
@@ -25,7 +25,7 @@ const getApplicationByReferenceNumberVariables = (referenceNumber: string, url:
    * if url includes LOSS_PAYEE_FINANCIAL_DETAILS_INTERNATIONAL_ROOT
    * then decryptFinancialInternational should be set to true
    */
-  if (url.includes(LOSS_PAYEE_FINANCIAL_DETAILS_INTERNATIONAL_ROOT)) {
+  if (url.includes(LOSS_PAYEE_FINANCIAL_DETAILS_INTERNATIONAL_ROOT) || url.includes(CHECK_YOUR_ANSWERS)) {
```

Comment:
I am just thinking is this better or 

`
if (url.includes(CHECK_YOUR_ANSWERS)) {
   decryptFinancialUk = true;
    decryptFinancialInternational = true;
}
`

Reply:
Good shout, updated.